### ISSUE TITLE
Kaihao Tian Personal Goals

### ISSUE THREAD

**Issue by KevsterT** [dartmouth-cs98/project-22f-farm-frenzy#124]
# Kaihao Tian | Personal Goals
*what does success in this class mean to you?*

I am able to show my work off in my portfolio, and it will give people some fun time.

## Challenges
*what are some challenges you foresee?*

We have a lot of small details to work on to get a smooth gameplay experience.
We need to decide if we move this game onto a server and run it on different devices.
We need to design a compact hub area for tutorials.

**Comment by nataliesvoboda** [comment]
I agree! There is a lot of design work to consider for the tutorials/onboarding that will be great to figure out early on in the term. Get going on polishing those sketches and user testing them before publishing to the main project.
